pkg/config/config.go

```diff
@@ -81,3 +91,20 @@ func validateAdditionalResources(resources []string) error {
 	}
 	return nil
 }
+
+// validateOutputFile checks if output file name is valid and if the
+// destination directory exists
+func validateOutputFile(outputFileName string) error {
+	if outputFileName == "" {
+		return fmt.Errorf("output file name can't be empty (use - for stdout)")
+	}
+
+	if outputFileName != "-" {
```

Comment:
shouldn't this return and error? I would catch this above

if outputFileName == ""  or outputFileName == "-"